corehq/form_processor/backends/sql/dbaccessors.py

```diff
@@ -248,6 +249,35 @@ def _get_forms_for_user(domain, user_id, state, ids_only=False):
             return [form.form_id for form in forms]
         return forms
 
+    @staticmethod
+    def get_all_forms_received_since(received_on_since=None, chunk_size=500):
+        start_from = received_on_since or datetime.min
```

Comment:
any reason to use both `start_from` and `received_on_since` could be confusing if someone things they're different things